codechain/keystore/HDStorageJson.py

```diff
@@ -0,0 +1,49 @@
+import binascii
+import os
+import uuid
+
+from Crypto.Cipher import AES
+
+from ..crypto import blake256
+from ..crypto import get_public_from_private
+from .Errors import ErrorCode
+from .Errors import KeystoreError
+from .keys import key_from_public_key
+from .KeyType import KeyType
+from .Pbkdf2Async import pbkdf2_async
+from .Types import SecretStorage
+
+
+async def encode(seed: str, passphrase: str, meta: str):
+    seed_hash = blake256(seed)
+    salt = os.urandom(32)
+    iv = os.urandom(16)
+
+    kdf = "pbkdf2"
+    kdf_params = {
+        "dklen": 32,
+        "salt": binascii.hexlify(salt).decode("ascii"),
+        "c": 262144,
+        "prf": "hmac-sha256",
+    }
+    derived_key = await pbkdf2_async(
+        passphrase, salt, kdf_params.get("c"), kdf_params.get("dklen"), "sha256"
+    )
+    cipher = AES.new(derived_key[:16], AES.MODE_EAX, iv=iv)
```

Comment:
There also `AES.MODE_CTR`. Why did you use `AES.MODE_EAX`?

Reply:
My mistake. Thx.